shell/platform/embedder/tests/embedder_test_context_gl.h

```diff
@@ -65,12 +77,13 @@ class EmbedderTestContextGL : public EmbedderTestContext {
 
   void* GLGetProcAddress(const char* name);
 
- protected:
-  virtual void SetupCompositor() override;
+ private:
+  // |EmbedderTestContext|
+  void SetSurface(SkISize surface_size) override;
 
-  void SetupCompositorUsingGLSurfaces();
+  // |EmbedderTestContext|
+  void SetupCompositor() override;
 
- private:
```

Comment:
This stuff is all private now instead of protected since there are no subclasses.